src/scss/_sofie-color-vars.scss

```diff
@@ -47,6 +49,8 @@ body {
   --gradient-black-rgba-with-06-opacity-color: rgba(0, 0, 0, 0.6);
   --gradient-black-rgba-with-08-opacity-color: rgba(0, 0, 0, 0.8);
   --gradient-green-rgba-with-04-opacity-color: rgba(0, 222, 99, 0.4);
+  --gradient-green-rgba-with-08-opacity-color: rgba(0, 222, 99, 0.8);
+  --gradient-red-rgba-with-08-opacity-color: rgba(255, 0, 0, 0.8);
```

Comment:
I can't find any usages of these variables. If that's true, they should be deleted again.